cyw43/src/runner.rs

```diff
@@ -116,6 +116,29 @@ where
         self.core_reset(Core::WLAN).await;
         assert!(self.core_is_up(Core::WLAN).await);
 
+        // bluetooth
+        //self.init_bluetooth().await;
+
+        // wifi
+        self.init_wifi().await;
+    }
+
+    pub(crate) async fn init_bluetooth(&mut self) {
+        // cybt_wait_bt_ready
+        loop {
+            let val = self
+                .bus
+                .bp_read32(CHIP.pmu_base_address + REG_BACKPLANE_BT_CTRL_REG_ADDR)
+                .await;
+            if val & BTSDIO_REG_BT_AWAKE_BITMASK == 1 {
```

Comment:
i can't tell why this is never being hit based on https://github.com/raspberrypi/pico-sdk/blob/master/src/rp2_common/pico_cyw43_driver/cybt_shared_bus/cybt_shared_bus.c#L112-L155

are they doing something different with the firmware upload/download?